### ISSUE TITLE
Default containers reappear when syncing from a new profile

### ISSUE THREAD

**Issue by saschanaz** [mozilla/multi-account-containers#1737]
- Multi-Account Containers Version: 6.2.5
- Operating System + Version: Windows 10 2004 (19041.207)
- Firefox Version: Nightly 78 and stable 76
- Other installed Add-ons + Version + Enabled/Disabled-Status: 

Extensions
Name 	Version 	Enabled 	ID
Amazon Smile Redirect	2.0.3	true	{aac801ed-5288-41e0-a937-7aa70207ab13}
Amazon.com	1.1	true	amazondotcom@search.mozilla.org
Bing	1.1	true	bing@search.mozilla.org
Dark Reader	4.9.8	true	addon@darkreader.org
DuckDuckGo	1.0	true	ddg@search.mozilla.org
Firefox Multi-Account Containers	6.2.5	true	@testpilot-containers
Google	1.0	true	google@search.mozilla.org
HTTPS Everywhere	2020.3.16	true	https-everywhere@eff.org
Image Search Options	3.0.12	true	{4a313247-8330-4a81-948e-b79936516f78}
Tampermonkey	4.10.6112	true	firefox@tampermonkey.net
To Google Translate	4.0.6	true	jid1-93WyvpgvxzGATw@jetpack
Twitter	1.0	true	twitter@search.mozilla.org
User-Agent Switcher	0.2.7	true	{75afe46a-7a50-4c6b-b866-c43a1075b071}
Wikipedia (en)	1.0	true	wikipedia@search.mozilla.org
Facebook Container	2.1.1	false	@contain-facebook

### Actual behavior

Default container "Work" reappears

### Expected behavior

It should only sync the renamed one

### Steps to reproduce
1. Rename one of the default containers "Work" to "Global"
2. Get a new Firefox profile, for example by installing Firefox on another device
3. Sync containers on it

**Comment by Solid-Ice8** [comment]
Repeat issue. Please read here > #1708

**Comment by saschanaz** [comment]
@Solid-Ice8 I don't think so as the sync itself works here. It's just that the default container revives from the new profile.
